kernel/tests/write.rs

```diff
@@ -251,3 +270,356 @@ async fn test_invalid_commit_info() -> Result<(), Box<dyn std::error::Error>> {
     ));
     Ok(())
 }
+
+// set the value at .-separated `path` in `values` to `new_value` at `index`
```

Comment:
I think this is a wrong comment. It _inserts/updates_ `new_value` into the object which must be at path `path`.

Reply:
oh yep oops had an old function that used an index and forgot to update :)